ui/tests/acceptance/secrets/backend/aws/aws-configuration-test.js

```diff
@@ -190,30 +196,27 @@ module('Acceptance | aws | configuration', function (hooks) {
       await runCmd(`delete sys/mounts/${path}`);
     });
 
-    test('it should save lease AWS configuration', async function (assert) {
-      assert.expect(3);
+    test('it does not save lease AWS configuration if root configuration errored on save', async function (assert) {
+      assert.expect(1);
```

Comment:
which assertion is expected here? Both say "test should fail" which isn't very clear. I remember us talking about this, but why did you opt to use an assertion here instead of throwing an error if a request is made to this endpoint?

If that's the case (and you still don't want to throw the error) can we change the assertion from "test should fail" (which isn't very clear) to "a request to this endpoint should NOT be made"

Reply:
Throwing an error is also what happens if I have `assert.false(true...` and it get's hit. It's just a different way of triggering an error. 

Happy to change the error message.

Reply:
I'll go ahead and change to throwing errors as I can see if the assert.false(true doesn't catch you're eye it's unclear it will throw an error.